climada/util/api_client.py

```diff
@@ -844,6 +844,44 @@ def get_litpop_default(self, country=None, dump_dir=SYSTEM_DIR):
             raise ValueError("country must be string or list of strings")
         return self.get_exposures(exposures_type='litpop', dump_dir=dump_dir, properties=properties)
 
+    def get_centroids(self, res_arcsec_land=150, res_arcsec_ocean=1800, extent=None, reg_id=None, country_iso3alpha=None,
+                      dump_dir=SYSTEM_DIR):
+        """Get global centroids
+
+        Parameters
+        ----------
+        res_land_arcsec : int
+            resolution for land centroids in arcsec. Default is 150
+        res_ocean_arcsec : int
+            resolution for ocean centroids in arcsec. Default is 1800
+        reg_id : int
+            region to filter according to region_id values
+        extent : tuple
```

Comment:
Should we put the default to have `min_lat = -60` and `max_lat = 60`? Also, the default is not explained. Is it the whole world?

Reply:
yes for now I put the whole world, I would keep it like that. Because for most hazards it makes sense to have the entire globe

Reply:
But is it cut at high/low latitudes or not? This would be important to know. The default extent could then be with +-60degrees?